static/js/components/AutoComplete.js

```diff
@@ -393,7 +405,8 @@ class AutoComplete extends Component {
       triggerUpdateOnFocus, // eslint-disable-line no-unused-vars
       openOnFocus, // eslint-disable-line no-unused-vars
       maxSearchResults,
-      dataSource,
+      dataSource, // eslint-disable-line no-unused-vars
+      menuHeight,
```

Comment:
I added `menuHeight` because we need to pass this number to the `VirtualScroll` method